server/services/v1/database/search_runner.go

```diff
@@ -63,6 +65,14 @@ func (runner *SearchQueryRunner) ReadOnly(ctx context.Context, tenant *metadata.
 	if err != nil {
 		return Response{}, ctx, err
 	}
+	if config.DefaultConfig.Search.LogFilter {
+		log.Error().
+			Str("tenant", tenant.Name()).
+			Str("project", db.Name()).
+			Str("collection", collection.Name).
+			Str("filter", string(runner.req.Filter)).
```

Comment:
We have to be careful with logging users filters. This could become a compliance issue.

Reply:
This is a temporary log to know if the user is using collection search with non-indexed filters.